### PR TITLE
feat: upgrade to js-ipfs 0.30

### PR DESCRIPTION
**Pull request by alanshaw**

### CODE REVIEW

File: add-on/src/lib/dir-view.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const filesize = require('filesize')
-const mainStyle = require('ipfs/src/http/gateway/dir-view/style')
+const mainStyle = require('ipfs-http-response/src/dir-view/style')
```
Comment (lidel) [inline add-on/src/lib/dir-view.js]: Does this mean we can remove `ipfs/src/http/gateway/dir-view/` from companion repo?

File: package.json [hunk 2]
```diff
@@ -87,9 +87,10 @@
     "file-type": "8.0.0",
     "filereader-pull-stream": "1.0.0",
     "filesize": "3.6.1",
-    "ipfs": "0.29.3",
-    "ipfs-api": "22.2.1",
+    "ipfs": "0.30.0",
+    "ipfs-api": "22.2.3",
     "ipfs-css": "0.5.2",
+    "ipfs-http-response": "^0.1.2",
```
Comment (lidel) [inline package.json]: `^` :upside_down_face:

File: add-on/src/lib/dir-view.js (hunk 1 above)
Comment (alanshaw) [reply add-on/src/lib/dir-view.js]: No, that's a path in the ipfs module

File: package.json (hunk 2 above)
Comment (alanshaw) [reply package.json]: 🤦‍♂️ every time

**Pull request merged**